frontend/src/App.tsx

```diff
@@ -75,7 +75,7 @@ export const FlowserRouter = () => {
     <BrowserRouter>
       <BrowserRouterEvents>
         <Switch>
-          <Route path={`/${routes.start}`} component={Start} />
+          <RouteWithLayout path={`/${routes.start}`} component={Start} />
```

Comment:
I think the reason we didn't use the `RouteWithLayout` here is that the navigation shouldn't be shown in the start page or project settings page, which is not the case after this change:

![Screenshot 2022-09-06 at 02 32 39](https://user-images.githubusercontent.com/36109955/188523825-744db554-cb4b-4e1b-8286-32e2b0307b7c.png)

I'm aware this may seem a bit unintentional, so maybe adding a comment could help clarify that?

Reply:
Oops, i changed this line, because the project settings page has the navbar on the new design,   I didn't notice that it affected the start page also. Will fix...